extensions/debuggee/src/org/exist/debuggee/ScriptRunner.java

```diff
@@ -68,8 +68,8 @@ public void stop() {
 	public void run() {
 		try {
 			final Database db = BrokerPool.getInstance();
-			
-            BrokerPool.registerStatusObserver(this);
+
+            db.addStatusObserver(this);
```

Comment:
@shabanovd You never unregistered it in the past... so I guess that is something for you to answer!?